lib/scrolling_sprite_component.dart

```diff
@@ -5,51 +5,25 @@ import 'dart:ui';
 import './scrolling_sprite.dart';
 
 class ScrollingSpriteComponent extends Component {
-  ScrollingSprite _sprite;
+  ScrollingSprite scrollingSprite;
```

Comment:
Just out of curiosity, why did you removed the private visibility here?

Reply:
Named param with underscore is not allowed. But we can do an interceptor there to change the name.

Reply:
No need, to be honest I think it is a good idea to expose this property.